### PR TITLE
2093 - Update hp-admin to work in HPOS

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import ./pkgs.nix {}, shell ? false }:
+{ pkgs ? import ./nixpkgs.nix {} }:
```
Comment (robbiecarlton) [inline default.nix]: @zo-el the original PR did't remove this `shell ? false` expression, that was something I did when resolving merge conflicts. Wasn't sure the right thing to do, maybe we want to keep that expression?

File: shell.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import ./pkgs.nix {}, shell ? false  }:
+{ pkgs ? import ./nixpkgs.nix {} }:
```
Comment (robbiecarlton) [inline shell.nix]: @zo-el - see comment on `default.nix` above

File: default.nix (hunk 1 above)
Comment (zo-el) [reply default.nix]: We do need that. It will give you access to `sim2h_server` cli

Comment (zo-el) [inline default.nix]:
```suggestion
{ pkgs ? import ./nixpkgs.nix {}, shell ? false }:
```

File: shell.nix (hunk 2 above)
Comment (zo-el) [inline shell.nix]:
```suggestion
{ pkgs ? import ./nixpkgs.nix {}, shell ? false }:
```

**Comment by robbiecarlton** [comment]
@zo-el ok, I incorporated the changes you suggested. LMK if this is good to merge

**Review by zo-el — APPROVED** [review]
Can we squash and merge this branch?

**Pull request merged**